plugins/node/opentelemetry-instrumentation-redis/package.json

```diff
@@ -59,6 +63,7 @@
     "cross-env": "7.0.3",
     "nyc": "17.1.0",
     "redis": "3.1.2",
+    "redis-v4": "npm:redis@4.7.1",
```

Comment:
reviewer note: this is needed for the `v4` tests. 
Since `redis` version is `3` by default, then `npm run compile` would fail when trying to import `RedisClientType` if not pulled into `node_modules` somehow.

When running the tests, `tav redis 4.7.1` would take care of updating the redis to make the test pass, but `compile` is run before that